knowledge-base/editor-add-text-templates.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Add text templates in Editor
+description: How to insert text templates in Editor?
+type: how-to
+page_title: Add text templates in Editor
+slug: editor-kb-add-text-templates
+position: 
+tags: editor, text, template, insert, add
+ticketid: 1589596
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Editor for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+Is there a way to insert templated text in the Editor?
+
+I want to have a tool that prefills various templates in the Editor. How to achieve this?
+
+## Solution
+
+You can achieve the desired result by creating a custom tool that programmatically changes the Editor value (inserts the desired templated text in the Editor).
```

Comment:
Every reference to a feature or API member should be a link. Use simpler sentences.

```suggestion
Create a [custom tool]({%slug editor-custom-tools%}) that programmatically changes the Editor value (inserts the desired templated text in the Editor).
```